src/System.Private.Xml.Linq/src/Resources/System.Private.Xml.Linq.rd.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <Directives xmlns="http://schemas.microsoft.com/netfx/2013/01/metadata">
   <Library Name="System.Xml.Linq">
-    <Assembly Name="System.Private.Xml.Linq">
+    <Assembly Name="System.Xml.Linq">
```

Comment:
@vitek-karas Yes, the assembly name is wrong. This is being tracked by #17557. 

I'm reverting the changes for `*Xml.Linq*` altogether. I verifed the changes in #17909 by building XmlSerializer tests on uapaot. The changes for XmlSerializer worked as expected. But It seems the tests didn't cover any *Xml.Linq* scenarios. For `*Xml.Linq*`, the verification I did was just making sure the warnings regarding to the rd.xml were gone. Therefore, I think it's better to leave the fix for `System.Private.Xml.Linq.rd.xml` to the person who would work on #17557.